sdk/core/azure-core-http-okhttp/src/test/java/com/azure/core/http/okhttp/OkHttpClientTests.java

```diff
@@ -214,8 +214,8 @@ public void testConcurrentRequests() throws NoSuchAlgorithmException {
             // Thread.currentThread().getName()))
             .map(nbb -> (long) nbb.bb.limit())
             .reduce(Long::sum)
-            .subscribeOn(reactor.core.scheduler.Schedulers.newElastic("io", 30))
-            .publishOn(reactor.core.scheduler.Schedulers.newElastic("io", 30));
+            .subscribeOn(Schedulers.newBoundedElastic(30, 1024, "io"))
+            .publishOn(Schedulers.newBoundedElastic(30, 1024, "io"));
```

Comment:
Why do we have both `subscribeOn` and `publishOn`? The last `publishOn` always wins over the `subscribeOn`.

Reply:
I'm not entirely certain as these tests have been around for a while, I'll clean up instances of this during the next round of changes.